src/api-reference/common/event-pubsub/travel-search-event.markdown

```diff
@@ -107,7 +107,7 @@ Name|Type|Format|Description
 Name|Type|Format|Description
 ---|---|---|---
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. <br> *NOTES:* </br>In cases where the individual is booking on behalf of a guest, a traveler without a Concur Travel profile, this is the user performing the search (e.g. the arranger).  In cases where the individual is booking on behalf of a traveler that does have a Concur profile (e.g. an arranger), this will be the arranger's ID and not the traveler.
```

Comment:
Update "on behalf of a guest, a traveler without a Concur Travel profile,"
to
"on behalf of a guest (a traveler without a Concur Travel profile),"
and update
"traveler that does have a Concur profile"
to
"traveler that does have a Concur Travel profile"